mantidproject/mantid
# Write out junit-style xml for Python unit tests

**Description of work**

Adds a wrapper test runner that forces the Python unit tests to write out their results to a junit-style
file that Jenkins can parse. This will avoid developers having to dig through the console when a Python
unit test fails.

The `xmlrunner` has been made into an external project.

**To test:**

Code review.

Run some of the Python unit tests with `ctest` and you should now see new `.xml` files for the tests in the `<build>/bin/Testing` directory.

**No issue**

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
